### PR TITLE
Template changes 202403.1.0

### PR DESCRIPTION
**Pull request by nnaik-conga**
End to end testing is done for the changes pushed to this pr.

### CODE REVIEW

**Review by dboppana-conga — APPROVED** [review]

File: src/environments/environment.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Configuration } from '@congarevenuecloud/core';
 
 export const environment: Configuration = {
-  production: true,
+  production: false,
```
Comment (vbhan-conga) [inline src/environments/environment.ts:4]: Why is this change required ?

Comment (nnaik-conga) [reply src/environments/environment.ts:4]: As this env file is not for production environment, the value was set back to false.
Since the customer can change this flag, I'll set it back to true.

**Review by vbhan-conga — APPROVED** [review]
